src/components/MegadraftEditor.js

```diff
@@ -198,7 +198,8 @@ export default class MegadraftEditor extends Component {
             plugins: this.plugins,
             editorState: this.props.editorState,
             readOnly: this.state.readOnly,
-            onChange: this.onChange
+            onChange: this.onChange,
+            numberPlugins: 3,
```

Comment:
Plese, choose a more descriptive name for this props, and make it customizable via props.